department-of-veterans-affairs/va.gov-cms
# VACMS-3908 update clp panel layout and text

## Description
See #3908 

## Testing done
Visual  / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/104249207-f3326500-5438-11eb-8350-354c3c126e7f.png)
![image](https://user-images.githubusercontent.com/2404547/104249216-f75e8280-5438-11eb-922f-1a1a8bdbe0e0.png)

## QA steps
 **As an administrator, go to `node/15051/edit` and visually verify that:**
-  Vertical tabs have been replaced by collapsible fieldsets
-  Hero banner fieldgroup is open by default, the rest are all closed.
-  Fieldgroups have help text
-  Collapsible fieldset text is not all in caps
-  Boolean TRUE labels all read "Enable this page segment" and have no field description.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
